provision/vvv-init.sh

```diff
@@ -1,31 +1,75 @@
+mysql -u root --password=root -e "GRANT ALL PRIVILEGES ON ${DB_NAME}.* TO wp@localhost IDENTIFIED BY 'wp';"
 echo -e "\n DB operations done.\n\n"
 
 # Nginx Logs
 mkdir -p ${VVV_PATH_TO_SITE}/log
 touch ${VVV_PATH_TO_SITE}/log/error.log
 touch ${VVV_PATH_TO_SITE}/log/access.log
 
-cd ${VVV_PATH_TO_SITE}
-
-noroot composer install
-
-cd ${VVV_PATH_TO_SITE}/wp-content/plugins/yoast-acf-analysis
-noroot composer install
-noroot npm install
+# Install and configure the latest stable version of WordPress
+if [[ ! -f "${VVV_PATH_TO_SITE}/public_html/wp-load.php" ]]; then
+    echo "Downloading WordPress..."
+	noroot wp core download --version="${WP_VERSION}"
+    echo "Downloading WordPress complete"
+fi
 
-if [[ ! -d "${VVV_PATH_TO_SITE}/wp" ]]; then
-  cp ${VVV_PATH_TO_SITE}/provision/.env.js ${VVV_PATH_TO_SITE}/wp-content/plugins/yoast-acf-analysis/tests/js/system/.env.js
+if [[ ! -f "${VVV_PATH_TO_SITE}/public_html/wp-config.php" ]]; then
+  echo "Configuring WordPress Stable..."
+  noroot wp core config --dbname="${DB_NAME}" --dbuser=wp --dbpass=wp --quiet --extra-php <<PHP
+define( 'WP_DEBUG', true );
+PHP
+  echo "Configuring WordPress complete"
 fi
 
 if ! $(noroot wp core is-installed); then
-    noroot wp core install --url=yoast-acf-analysis.vvv.dev --title="Yoast ACF Analysis Development" --admin_name=admin --admin_email="admin@local.dev" --admin_password="password" --skip-email
-    noroot wp plugin activate --all
-    # This is needed because the test relies on at least one attachment existing
-    noroot wp media import https://s.w.org/about/images/logos/wordpress-logo-notext-rgb.png
+  echo "Installing WordPress Stable..."
+
+  if [ "${WP_TYPE}" = "subdomain" ]; then
+    INSTALL_COMMAND="multisite-install --subdomains"
+  elif [ "${WP_TYPE}" = "subdirectory" ]; then
+    INSTALL_COMMAND="multisite-install"
+  else
+    INSTALL_COMMAND="install"
+  fi
+
+  noroot wp core ${INSTALL_COMMAND} --url="${DOMAIN}" --quiet --title="${SITE_TITLE}" --admin_name=admin --admin_email="admin@local.test" --admin_password="password" --skip-email
+
+  # This is needed because the test relies on at least one attachment existing
+  noroot wp media import https://s.w.org/about/images/logos/wordpress-logo-notext-rgb.png
+  echo "WordPress installation complete"
+
+  echo "Installing necessary plugins..."
+  cd ${VVV_PATH_TO_SITE}
+  noroot composer install -q
+  echo "Installing necessary plugins complete"
+
+  echo "Configuring Yoast ACF Analysis..."
+  cp ${VVV_PATH_TO_SITE}/provision/.env.js ${VVV_PATH_TO_SITE}/public_html/wp-content/plugins/yoast-acf-analysis/tests/js/system/.env.js
+  cp ${VVV_PATH_TO_SITE}/public_html/wp-content/plugins/yoast-acf-analysis/tests/js/system/nightwatch.conf.example.js ${VVV_PATH_TO_SITE}/public_html/wp-content/plugins/yoast-acf-analysis/tests/js/system/nightwatch.conf.js
+
+  sed -i "s#{{DOMAIN}}#http://${DOMAINS}#" "${VVV_PATH_TO_SITE}/public_html/wp-content/plugins/yoast-acf-analysis/tests/js/system/nightwatch.conf.js"
```

Comment:
I think we should also automatically do

```
cd ${VVV_PATH_TO_SITE}/public_html/wp-content/plugins/yoast-acf-analysis
composer install
npm install
```
instead of adding [this note](https://github.com/kraftner/yoast-acf-analysis-development/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R20). Without that the plugin won't init anyway because of [the lack of the vendor folder](https://github.com/Yoast/yoast-acf-analysis/blob/6f5373b93cb392a210282da21f5a304472f9b5e6/yoast-acf-analysis.php#L49).